autogen/main/README.md

```diff
@@ -199,6 +199,8 @@ The node_pools variable takes the following parameters:
 {% endif %}
 | min_count | Minimum number of nodes in the NodePool. Must be >=0 and <= max_count. Should be used when autoscaling is true | 1 | Optional |
 | name | The name of the node pool |  | Required |
+| network_config | Configuration for Adding Pod IP address ranges to the node pool. |  | Optional |
```

Comment:
Since only `pod_range` is used now, could we collapse this into being the only variable? So `network_config` doesn't need to be supplied at all?

Reply:
future proofing? what there will an additional var there?

Reply:
YAGNI - we shouldn't add complexity for hypothetical gains.

Reply:
Fair enough. changed.